src/Invictus.Testing.LogicApps/LogicAppAuthentication.cs

```diff
@@ -86,18 +101,20 @@ public static LogicAppAuthentication UsingAccessToken(string subscriptionId, str
         /// <param name="subscriptionId">The ID that identifies the subscription on Azure.</param>
         /// <param name="accessTokenKey">The secret key to use to fetch access token from the secret provider. This will be used to call the Azure management API.</param>
         /// <param name="secretProvider">The provider to get the client secret; using the <paramref name="accessTokenKey"/>.</param>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="subscriptionId"/> or <paramref name="accessTokenKey"/> is blank.</exception>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="secretProvider"/> is blank.</exception>
         public static LogicAppAuthentication UsingAccessToken(string subscriptionId, string accessTokenKey, ISecretProvider secretProvider)
```

Comment:
Key is very confusing if you ask me - Is it the "access token" key, Azure Key Vault Key which is not the same as secret, etc etc.
```suggestion
        public static LogicAppAuthentication UsingAccessToken(string subscriptionId, string accessTokenSecretName, ISecretProvider secretProvider)
```

I would rename this already before we ship this.

Reply:
Not really what this PR is about, but I'll change it 😄 .